packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx

```diff
@@ -11,13 +11,18 @@ import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
 const WebsiteLinkCard = ({ link }) => {
   const linkStyle = { textTransform: 'lowercase' }
+  const hrStyle = {
+    height: '1px',
+    border: 'none',
+    backgroundColor: palette['silver']
+  }
   const label = new URL(link).host
   const { t } = useI18n()
 
   return (
     <Card>
       <SubTitle className="u-mb-1">{t('card.websiteLink.title')}</SubTitle>
-      <hr className="u-silver" />
+      <hr style={hrStyle} />
```

Comment:
You can use `react/MuiCozyTheme/Divider/index.jsx` for this kind of Divider.

For cases like that, it is not a good practice to use adhoc styling. If there is a divider here, there is a good likelihood that there will be other divider in other parts of the UI.

Reply:
I could not find this divider in https://docs.cozy.io/cozy-ui/react/#/Material-UI and when I try it, I get an error.